presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeWindowQueries.java

```diff
@@ -115,4 +153,44 @@ public void testRowNumberWithFilter()
         assertQuery("SELECT sum(rn) FROM (SELECT row_number() over() rn, * from orders) WHERE rn = 10");
         assertQuery("SELECT * FROM (SELECT row_number() over(partition by orderstatus order by orderkey) rn, * from orders) WHERE rn = 1");
     }
+
+    @Test
+    public void testFirstValue()
+    {
+        testWindowFunction("first_value(orderkey)", FunctionType.VALUE);
+        testWindowFunction("first_value(orderdate)", FunctionType.VALUE);
+        testWindowFunction("first_value(totalprice)", FunctionType.VALUE);
```

Comment:
The invocations with total_price are not needed for first_value, last_value and nth_value. Why did you add them ?

Reply:
The tests for different column types were added to identify any issues that might come up. It is removed now.